mindsworth/My-Diary-Frontend
# #162361773 Configured Eslint

#### What does this PR do?
Eslint configuration using Airbnb style guide.

#### Description of Task to be completed?
- Installed Eslint
- create .eslintrc file
- configured eslint airbnb style guide
- installed babel-eslint

#### How should this be manually tested?
```
1. Clone this repo
2. open it on a text editor and run `yarn install`
3. Once everything is done installing, your code should start linting.
```
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[ #162361773](https://www.pivotaltracker.com/story/show/162361773)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
